wdio.conf.e2e.js

```diff
@@ -11,5 +20,40 @@ export const config = {
   services: [
     ...base.services,
     [WebpackDevServerService, {}]
-  ]
+  ],
+  // chrome configuration for download testing
+  capabilities: [{
+    browserName: 'chrome',
+    'goog:chromeOptions': {
+      prefs: {
+        'directory_upgrade': true,
+        'prompt_for_download': false,
+        'download.default_directory': downloadDir
+      },
+      args: [
+        'headless',
+        'disable-gpu'
+      ]
+    }
+  },
+  {
+    browserName: 'firefox'
+  },
+  {
+    browserName: 'safari'
+  }],
+  onPrepare: function (config, capabilities) {
+    // make sure download directory exists
+    if (!fs.existsSync(downloadDir)){
+        // if it doesn't exist, create it
+        fs.mkdirSync(downloadDir);
+    }
+  },
+  onComplete: function() {
+    console.log(downloadDir)
+    fs.rmSync(downloadDir, {
+      recursive: true,
+      force: true
+    });
+  }
```

Comment:
I was taking a look at this locally and realized there are a few issues:
- This overrides the browser configuration from `wdio.conf.base.js` and breaks support for the `BROWSERS` environment variable. It needs to move there: https://github.com/Mr0grog/google-docs-to-markdown/blob/a01cbb3d6f0640ca57022bc66b9345903eb323c7/wdio.conf.base.js#L3-L21 (I know the download folder is specific to the e2e tests, but it won’t hurt the unit tests to configure it for them, too.)

- I was surprised this worked for Firefox and Safari without any special configuration, but then realized that, because the browsers each run all the tests in parallel, the Firefox and Safari tests are actually just reading the output from the Chome test! Each browser needs its own directory.

I’ll take a look at fixing these.

Reply:
Arghhh my apologies! Nothing like a pile of work wrapped up with a PR bow from a stranger haha. sorry!

Reply:
No worries! I strongly suspected this was going to be one of those features that's 10 times harder to test than to implement. You got a lot of the initial stuff already done, so 👍